src/api/hooks/useSerialPublicationSearch.ts

```diff
@@ -0,0 +1,27 @@
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { keepSimilarPreviousData } from '../../utils/searchHelpers';
+import { searchForSerialPublications } from '../publicationChannelApi';
+
+interface SerialPublicationSearchParams {
+  searchMode: 'journal' | 'series';
+  searchTerm: string;
+  size: number;
+  year?: string;
+}
+
+export const useSerialPublicationSearch = ({
+  searchMode,
+  searchTerm,
+  year = new Date().getFullYear().toString(),
+  size,
+}: SerialPublicationSearchParams) => {
+  const { t } = useTranslation();
+  return useQuery({
+    queryKey: ['serialPublicationSearch', searchTerm, year, size],
+    enabled: searchTerm.length > 3,
```

Comment:
```suggestion
    enabled: searchTerm.length >= 3,
```
Er det en grunn til at man må skrive fire tegn før søket starter? Er tre kanskje mer naturlig?

Reply:
Det er en begrensing i APIet https://api.dev.nva.aws.unit.no/publication-channels-v2/serial-publication?query=nat